v2/index.js

```diff
@@ -0,0 +1,33 @@
+import cors from "cors";
+import dotenv from "dotenv";
+import express from "express";
+import mongoose from "mongoose";
+import forgotPasswordRoute from "./routes/forgotPasswordRoute.js";
+import resetPasswordRoute from "./routes/resetPasswordRoute.js";
+import signInRoute from "./routes/signInRoute.js";
+import signUpRoute from "./routes/signUpRoute.js";
+
+dotenv.config();
+
+const app = express();
+
+//middlewares
+app.use(cors());
+app.use(express.json());
+
+//routes
+app.use("/signup", signUpRoute); //sign up route
```

Comment:
Remove all the inline comments. For this project we will not use inline comments.